cnxauthoring/tests/test_functional.py

```diff
@@ -3300,14 +3311,14 @@ def test_publish_derived_from_binder(self):
 
         publication_info = response.json
         publication_id = publication_info['publication']
-        self.assertEqual(publication_info['state'], 'Done/Success')
+        self.assertEqual(publication_info['state'], 'Waiting for moderation')
```

Comment:
This is the commit that fixes the unit tests:
```
Update test setup to let user1 publish without moderation …

First time publishers won't be able to publish documents until a
moderator approves the publication.  Poke the archive / publishing
database to let user1 publish without moderation.
```

Users only get "Waiting for moderation" for their first publication as far as I understand.
I looked at how publishing decides whether a user needs moderation or not, and I added some code in the test set up so user1 can publish.  We use user1 in most of our tests so most of the publications remain "Done/Success".

In test_publish_derived_from_binder, we're logged in as e5a07af6-09b9-4b74-aa7a-b7510bee90b8.  It's a first time publisher, so the state is "Waiting for moderation".

Does this answer your questions?